frontend/src/concepts/userSSAR/useKebabAccessAllowed.ts

```diff
@@ -0,0 +1,53 @@
+import { IAction } from '@patternfly/react-table';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import { useAccessAllowed } from '~/concepts/userSSAR/useAccessAllowed';
+
+const tooltipArgs = (text: string): Partial<IAction> => ({
+  isAriaDisabled: false,
+  isDisabled: false,
```

Comment:
```suggestion
  isAriaDisabled: true,
```

i mean this is broken in pf6 so this is fine, but it should be this

Reply:
PF is aware of it https://github.com/patternfly/patternfly-react/issues/11358